framework/caching/Cache.php

```diff
@@ -187,15 +202,15 @@ public function multiGet($keys)
                 if ($this->serializer === false) {
                     $results[$key] = $values[$newKey];
                 } else {
-                    $value = $this->serializer === null ? unserialize($values[$newKey])
-                        : call_user_func($this->serializer[1], $values[$newKey]);
+                    $value = $this->serializer === null ? unserialize($values[$newKey]) : call_user_func($this->serializer[1], $values[$newKey]);
 
                     if (is_array($value) && !($value[1] instanceof Dependency && $value[1]->isChanged($this))) {
                         $results[$key] = $value[0];
                     }
                 }
             }
         }
+        $this->endProfile($keyMap, __METHOD__);
```

Comment:
In `get()` we currently only profile the interaction with the cache backend, here we also include deserialisation. This should be consistent, so endProfile call should be moved to line 198.